features/interactivity.yml

```
@@ -0,0 +1,3 @@
+name: interactivity
+description: The CSS `interactivity` property specifies whether an element and its descendants are inert or not.
```

```suggestion
description: The `interactivity: inert` CSS declaration makes an element and its descendants inert, like when using the `inert` HTML attribute. Inert elements can't be focused or clicked, their text can't be selected or found using the browser's find-in-page feature.
```

I like adding the description of inert here. I'd suggest adding the full declaration here as well.